### DISCUSSION TITLE
Documentation Site Consideration: Figma Guides

### DISCUSSION THREAD

**Discussion by bennettcc** [NCIOCPL/ncids#1377] [NCIDS Launch]
Submitted by Anna during Doc site review:

How should we be referring to the Figma Guide: “Figma Guide” singular in navigation v “Figma Guides” plural in the page content.

**Comment by blilianyu** [comment]
It is one guide. On page content, there is not plural usage. 
 
cc: @monika-jaeger

**Comment by laurelthrash** [comment]
This item has been addressed as referenced in the above comment. Closing this discussion. Cc: @blilianyu @monika-jaeger
